src/tribol/interface/tribol.cpp

```diff
@@ -96,9 +101,9 @@ void setPenaltyOptions( int couplingSchemeIndex, PenaltyConstraintType pen_enfrc
    CouplingSchemeManager& csManager = CouplingSchemeManager::getInstance();
 
    // check to see if coupling scheme exists
-   SLIC_ERROR_IF( !csManager.hasCoupling( couplingSchemeIndex ), 
-                  "tribol::setPenaltyOptions(): call tribol::registerCouplingScheme() " <<
-                  "prior to calling this routine." );
+   SLIC_ERROR_ROOT_IF( !csManager.hasCoupling( couplingSchemeIndex ), 
```

Comment:
Note: The `_ROOT_` calls only work when a specific API call is invoked -- (`slic::setIsRoot()`)
See: https://github.com/LLNL/axom/blob/ca7e58ff86f5f16c4c1f57f3042bd911905adb13/src/axom/slic/interface/slic.hpp#L62-L68

Reply:
I added this to `tribol::initialize()` (I believe correctly).

Reply:
I removed this. We agreed that for now the host-code is responsible for all initialization, calls to setIsRoot() and finalize.